packages/cli/.nycrc

```diff
@@ -8,6 +8,7 @@
   "exclude": [
     "src/cmds/preview/depsChecker/**/*",
     "src/cmds/preview/preview.ts",
+    "src/cmds/preview/commonUtils.ts",
```

Comment:
Seems no change in `commonUtils`, why adding this?

Reply:
The refactor removes some code in constants.js, which causes decreases in code coverage.

Reply:
Add some test cases of commonUtil